config/initializers/redis.rb

```diff
@@ -0,0 +1,2 @@
+$REDIS_URL = ENV['REDIS_URL'] || 'redis://localhost:6379'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.